xclog_parser.py

```diff
@@ -109,6 +109,11 @@ def parse_compile_swift_module(self, line: str):
         echo(f"CompileSwiftModule {module['module_name']}")
         return module
 
+    def parse_swift_error(self, line: str):
+        if not line.startswith('/'): return
+        if not line.__contains__('error'): return
```

Comment:
Use more robust tags, and consider whether to add warnings, such as regular: r":\s(error|warning):"